@xen-orchestra/lite/src/stories/web-core/ui/circle-progress-bar/ui-circle-progress-bar.story.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('value').num().preset(75).required().widget(),
+      prop('max-value').num().default(100).preset(100).widget(),
```

Comment:
The prop is required, plus there is no default value (only `preset` is enough):
```suggestion
      prop('max-value').num().required().preset(100).widget(),
```